c10/core/impl/LocalDispatchKeySet.h

```diff
@@ -93,6 +93,21 @@ class C10_API ExcludeDispatchKeyGuard {
   bool prev_state_;
 };
 
+class C10_API ExcludeDispatchKeySetGuard {
```

Comment:
Maybe we can just get rid of ExcludeDispatchKeyGuard now, only support sets.

Reply:
Went ahead and did this, for Include as well.

Reply:
#42658